### PR TITLE
gau: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/gau.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`gau --providers {{wayback|commoncrawl|otx|urlscan}} {{ejemplo.com}}`
+
+- Busca las URL de varios proveedores:
+
+`gau --providers {{wayback,otx,...}} {{ejemplo.com}}`
+
+- Busca las URLs dentro de un intervalo de fechas específico:
+
+`gau --del {{AAAAMM}} --to {{YYYYMM}} {{ejemplo.com}}`
```
Comment (tricantivu) [inline pages.es/common/gau.md:32]:
```suggestion
`gau --from {{AAAAMM}} --to {{YYYYMM}} {{ejemplo.com}}`
```

AFAIK, the translated [option string is not valid](https://github.com/lc/gau/blob/d556483b1282330af51aa2b5e5a906d4520ceacd/runner/flags/flags.go#L117)

File: pages.es/common/gau.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Escribe los resultados de salida en un archivo:
+
+`gau {{ejemplo.com}} --o {{ruta/a/urls_encontradas.txt}}`
+
+- Busca las URLs de un solo proveedor específico:
+
+`gau --providers {{wayback|commoncrawl|otx|urlscan}} {{ejemplo.com}}`
+
+- Busca las URL de varios proveedores:
```
Comment (tricantivu) [inline pages.es/common/gau.md:26]:
```suggestion
- Busca las URLs de varios proveedores:
```

File: pages.es/common/gau.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Trae URLs de varios dominios:
+
+`gau {{dominio1 dominio2 ...}}`
+
+- Trae todas las URLs de varios dominios desde un archivo de entrada, ejecutando varios subprocesos:
+
+`gau --threads {{4}} < {{ruta/a/dominios.txt}}}`
+
+- Escribe los resultados de salida en un archivo:
```
Comment (tricantivu) [inline pages.es/common/gau.md:18]:
```suggestion
- Escribe los resultados de en un archivo:
```

File: pages.es/common/gau.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Obtiene todas las URL de un dominio de Open Threat Exchange de AlienVault, de Wayback Machine, de Common Crawl y de URLScan:
+
+`gau {{ejemplo.com}}`
+
+- Trae URLs de varios dominios:
+
+`gau {{dominio1 dominio2 ...}}`
+
+- Trae todas las URLs de varios dominios desde un archivo de entrada, ejecutando varios subprocesos:
```
Comment (tricantivu) [inline pages.es/common/gau.md:14]:
```suggestion
- Obtén todas las URLs de varios dominios en un archivo de entrada, ejecutando varios subprocesos:
```

File: pages.es/common/gau.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# gau
+
+> Obtiene todas las URLs: trae las URLs conocidas de Open Threat Exchange de AlienVault, de Wayback Machine y de Common Crawl para cualquier dominio.
+> Más información: <https://github.com/lc/gau>.
+
+- Obtiene todas las URL de un dominio de Open Threat Exchange de AlienVault, de Wayback Machine, de Common Crawl y de URLScan:
```
Comment (tricantivu) [inline pages.es/common/gau.md:6]:
```suggestion
- Obtiene todas las URLs de un dominio de Open Threat Exchange de AlienVault, Wayback Machine, Common Crawl y URLScan:
```

File: pages.es/common/gau.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+`gau {{ejemplo.com}}`
+
+- Trae URLs de varios dominios:
```
Comment (tricantivu) [inline pages.es/common/gau.md:10]:
```suggestion
- Obtén URLs de varios dominios:
```

File: pages.es/common/gau.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# gau
+
+> Obtiene todas las URLs: trae las URLs conocidas de Open Threat Exchange de AlienVault, de Wayback Machine y de Common Crawl para cualquier dominio.
```
Comment (tricantivu) [inline pages.es/common/gau.md:3]:
```suggestion
> Obtén todas las URLs: obtén las URLs conocidas de Open Threat Exchange de AlienVault, Wayback Machine y Common Crawl para cualquier dominio.
```

File: pages.es/common/gau.md (hunk 3 above)
Comment (kant) [reply pages.es/common/gau.md:18]: @tricantivu missing translated word

**Review by kbdharun — APPROVED** [review]

File: pages.es/common/gau.md (hunk 3 above)
Comment (tricantivu) [reply pages.es/common/gau.md:18]: I made some changes

**Review by tricantivu — APPROVED** [review]

**Pull request merged**